YELLOW_BELT/week_1/matrix_summ.cpp

```diff
@@ -0,0 +1,107 @@
+#include <fstream>
+#include <iostream>
+#include <stdexcept>
+#include <vector>
+
+class Matrix {
+ public:
+  int num_rows_;
+  int num_columns_;
+  int** arr;
+  std::vector<std::vector<int> > elements_;
```

Comment:
@Ypostolov We discussed to do it via the array, not vector

Reply:
Fixed in c049e8ef94431348aec0ca0e79069a828d8fc1ba.
Please check.